playbooks/roles/airship-deploy-ucp/templates/airship.yaml.j2

```diff
@@ -78,6 +78,31 @@ data:
     subpath: ingress
   dependencies:
     - osh-helm-toolkit
+
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-postgresql
+data:
+  chart_name: ucp-postgresql
+  release: ucp-postgresql
+  namespace: ucp
+  wait:
+    timeout: 1800
+    labels:
+      release_group: airship-ucp-postgresql
+  values:
+    pod:
+      security:
+        server:
+          runAsUser: 0
```

Comment:
Is root user the permanent solution? If not, can we add a todo comments?

Reply:
At the moment, we don't know. Ideally we need to figure out why its issue for us with caasp node.
May be we need to investigate more later.

Is there standard convention for TODO here to follow.

Reply:
I am not aware. I followed the existing #TODO comments in the socok8s.